docs/readme.md

```diff
@@ -78,6 +89,38 @@ Installing dependencies from Pipfile.lock (65135d)…
   🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 14/14 — 00:00:03
 ```
 
+## LinkCheck
+After preparing your changes and reviewing the rendered HTML in a web browser, you should check to make sure that the documentation does not have any broken links. To do this:
```

Comment:
Could this be automated in precheckin?

Reply:
From what I learned as of now only a static URL can be added to Github action. We may have to find a way to build the docs, then serve it on local URL and run linkcheck. Created a JIRA  https://delphix.atlassian.net/browse/DLPX-84560 for future references.

Also, tried multiple ways to add linkcheck to pre-commit but seems like it will need more research.